apps/aklite-offline/cmds.h

```diff
@@ -88,7 +99,9 @@ class InstallCmd : public Cmd {
 
 class RunCmd : public Cmd {
  public:
-  RunCmd() : Cmd("run", _options) {
+  RunCmd()
+      : Cmd("run", "Finalizes the installation of a target that required a reboot, starting the updated apps",
```

Comment:
Is 'run' not required for an apps only update anymore?

Reply:
You're right, we did not change that behavior. I've updated the command description.